src/pal/inc/unixasmmacrosarm.inc

```diff
@@ -146,17 +131,33 @@ C_FUNC(\Name\()_End):
 
         __PWTB_StackAlloc = __PWTB_TransitionBlock
 
+	// Make the stack operation compatible with GDB and LIBUNWIND
+	PROLOG_PUSH		"{r7, lr}"
```

Comment:
The stack layout created by these macros has to match `TransitionBlock` in `vm\callingconvention.h`.

Reordering of the callee saved registers - to save r7 and lr next to each other - should be fine.

Moving argument registers around will have pretty non-trivial consequences. Would it be possible to keep them where they are?